templates/cloud/index.html

```diff
@@ -1,250 +1,364 @@
+{% endblock meta_description %}
 
-{% block meta_copydoc %}https://docs.google.com/document/d/1DJZFADvRr_hrHzRWJrUTJxAM2ocZJ_ZWEnQVdKi6JBE/edit#{% endblock meta_copydoc %}
+{% block meta_image %}
+  https://assets.ubuntu.com/v1/b0824086-twitter_wide_banner+%288%29.jpeg
+{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1DJZFADvRr_hrHzRWJrUTJxAM2ocZJ_ZWEnQVdKi6JBE/edit#
+{% endblock meta_copydoc %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
 
 {% block content %}
-<section class="p-strip--suru-topped is-deep">
-  <div class="row">
-    <div class="col-8">
-      <h1>Private cloud vs hybrid cloud, multicloud and more</h1>
-      <p class="p-heading--4">Choose the cloud architecture that suits you best</p>
-      <p>There is no one size fits all cloud architecture. Developing the optimum cloud strategy requires evaluating your business needs and aligning them with the different solutions available.</p>
-      <p>Canonical fully supports public clouds and provides its own <a href="https://canonical.com/solutions/infrastructure">infrastructure solutions</a> for <a href="/openstack/consulting">private cloud implementation</a> and <a href="/openstack/managed">management</a>, as well as workload orchestration in hybrid cloud and multicloud environments.</p>
-      <p>
-        <a class="p-button--positive" href="/engage/cloud-cost-optimisation-best-practices">Optimise your cloud infrastructure</a>
-      </p>
-    </div>
-    <div class="col-4 u-hide--small u-vertically-center u-align--center">
-      {{
-        image(
-         url="https://assets.ubuntu.com/v1/ab87e245-Private-v-Public-Mulit-Cloud.svg",
-         alt="Private vs public cloud",
-         width="350",
-         height="300",
-         hi_def=True,
-         loading="auto",
-        ) | safe
-      }}
-    </div>
-  </div>
-</section>
+  <section class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
+        <h1>Private cloud vs hybrid cloud, multicloud and more</h1>
+      </div>
+      <div class="col">
+        <p class="p-heading--2">Choose the cloud architecture that suits you best</p>
+        <p>
+          There is no one size fits all cloud architecture. Developing the optimum cloud strategy requires evaluating your business needs and aligning them with the different solutions available.
+        </p>
+        <p class="p-section--shallow">
+          Canonical fully supports public clouds and provides its own <a href="https://canonical.com/solutions/infrastructure">infrastructure solutions</a> for <a href="/openstack/consulting">private cloud implementation</a> and <a href="/openstack/managed">management</a>, as well as workload orchestration in hybrid cloud and multicloud environments.
+        </p>
```

Comment:
```suggestion
        <div  class="p-section--shallow">
          <p>
            Canonical fully supports public clouds and provides its own <a href="https://canonical.com/solutions/infrastructure">infrastructure solutions</a> for <a href="/openstack/consulting">private cloud implementation</a> and <a href="/openstack/managed">management</a>, as well as workload orchestration in hybrid cloud and multicloud environments.
          </p>
        </div>
```
I wasn't sure about this one, so I reached out to the Vanilla team for there input. Basically, they told me we should avoid putting structural/component classes (ie. p-section) on semantic elements (ie. <p>). [See the conversation for more details](https://chat.canonical.com/canonical/pl/yjjhnfdumtf5zm9ften6pghxeo).